website/content/docs/drivers/raw_exec.mdx

```diff
@@ -159,6 +162,15 @@ config {
 }
 ```
 
+- `denied_envvars` - (Optional) Passes a list of environment variables to be
+  scrubbed from all task environments. Supports simple globbing, "*" wildcard
+  accepted as prefix and/or suffix.
+
+```
+config {
+  denied_envvars = "["AWS_SECRET_KEY", "*_TOKEN]
+}
```

Comment:
I can't figure out how to add the triple backtick as a suggestion, but we need to close this code block so that the rest of the page isn't in the example. See https://nomad-git-f-net-12348-rawexecdenylist-hashicorp.vercel.app/nomad/docs/drivers/raw_exec#denied_envvars-1 for the rendered version.